recipe/meta.yaml

```
@@ -1,39 +1,47 @@
 {% set name = "mkdocs-material-extensions" %}
-{% set version = "1.0.3" %}
+{% set version = "1.1" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: bfd24dfdef7b41c312ede42648f9eb83476ea168ec163b613f9abd12bbfddba2
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mkdocs_material_extensions-{{ version }}.tar.gz
+  sha256: 96ca979dae66d65c2099eefe189b49d5ac62f76afb59c38e069ffc7cf3c131ec
```

```suggestion
  sha256: 9c003da71e2cc2493d910237448c672e00cefc800d3d6ae93d2fc69979e3bd93
```
https://pypi.org/project/mkdocs-material-extensions/1.1.1/#copy-hash-modal-4ff14440-53c7-4e5c-a24f-f2b42fb00435